app/views/articles/show.html.erb

```diff
@@ -231,8 +231,13 @@
   <% end %>
 <% end %>
 
-
-<div class="mod-actions-menu"><iframe src="<%= "#{@article.path}/mod" %>"></iframe></div>
+<!-- Mod Actions Menu & Trigger Icon-->
+<% if current_user.id != @article.user_id && current_user.trusted %>
```

Comment:
This isn't going to work because this page will be viewed as cached HTML and `current_user` will not be available in production.

I think this should be implemented in the JavaScript similar to where it was above.

I'd say keep this if statement to include `<% if user_signed_in? %>` because we do have knowledge on the server of whether the user is logged in or not, but then hide the button and reveal it via JavaScript if the user is trusted.

Reply:
oh wow, thanks Ben! I will do this right away

Reply:
will do 👍